static/scripts/secret.ts

```diff
@@ -45,6 +46,7 @@ interface IConf {
 }
 
 const defaultConf: IConf = {
+  'chain-id': 1,
```

Comment:
shouldn't chainId be an array or what's the purpose of it?
how should it follow through?

Reply:
we add `chain-id` param so that partner could select which chain to use for payments

more info [here](https://github.com/ubiquity/ubiquibot/issues/287)